### DISCUSSION TITLE
install tailwind css in vintage way.

### DISCUSSION THREAD

**Discussion by JoyDhar32** [tailwindlabs/tailwindcss#4104] [Help] [answered]
![Capture](https://user-images.githubusercontent.com/54507249/115149596-ae951600-a086-11eb-8050-21eae05765e9.PNG)
Hello!!!
This is my first discussion and I am confused to install tailwind css. There are different documentation between V2.1.0 and V1.9.6

I have installed by following V1.9.6 documentation and checked package.json the version 2.1 is automatically installed. Is there will any issue if I follow installation documentation of V1.9?

Thanks.
![Capture](https://user-images.githubusercontent.com/54507249/115149585-9cb37300-a086-11eb-9377-610518e136f9.PNG)

**Comment by mcmire** [comment]
Probably not, but just curious, what is tripping you up with the v2.1 documentation?

**Reply by JoyDhar32** [reply]
@mcmire  Thanks for your comment. Actually I feel comfort v1 installation method rather than V2. But both process installing the update version. So I think I can follow the v2 documentation whatever installation process

**Comment by simonswiss** [comment, marked as answer]
You should be fine, that part is essentially the same. Is there a reason why you followed the v1 version installation guide instead of v2? Are you encountering any problem with the way you did it?

**Reply by JoyDhar32** [reply]
@simonswiss Thanks for your comment. Actually I feel comfort v1 installation. In V2 is confusing for me. But after  followed v1 installation process and checked package.json file here tailwind version 2 is installed. Both process installing the update version of tailwind. As I am going for a big project using tailwind so there is my confusion is all ok or not.
